package.json

```diff
@@ -10,6 +10,8 @@
     "@types/node": "^16.11.26",
     "@types/react": "^17.0.42",
     "@types/react-dom": "^17.0.14",
+    "buffer": "^6.0.3",
+    "lnurl-pay": "^0.2.3",
```

Comment:
did you end up getting this working? invoice isn't popping up it just goes to processing lightnight payment